pkg/kubelet/apis/config/types.go

```diff
@@ -400,9 +401,13 @@ type KubeletConfiguration struct {
 	// The format is <major>.<minor>, e.g.: '1.16'.
 	// The purpose of this format is make sure you have the opportunity to notice if the next release hides additional metrics,
 	// rather than being surprised when they are permanently removed in the release after that.
```

Comment:
Please also mention that the value under metrics.ShowHiddenMetricsForVersion has higher precedence